src/Item/Item.router.ts

```diff
@@ -182,21 +192,33 @@ export class ItemRouter extends Router {
       )
     }
 
-    const [dbItems, remoteItems, dbTPItems, itemCurations] = await Promise.all([
-      Item.findNonThirdPartyItemsByOwner(eth_address),
+    const [allItems, remoteItems, itemCurations] = await Promise.all([
+      this.itemService.findAllItemsForAddress(
+        eth_address,
+        limit,
+        page && limit ? getOffset(page, limit) : undefined
+      ),
       collectionAPI.fetchItemsByAuthorizedUser(eth_address),
-      this.itemService.getTPItemsByManager(eth_address),
       ItemCuration.find<ItemCurationAttributes>(),
     ])
 
+    const { items: dbItems, tpItems: dbTPItems } = this.itemService.splitItems(
+      allItems
+    )
+
     const [items, tpItems] = await Promise.all([
       Bridge.consolidateItems(dbItems, remoteItems),
       Bridge.consolidateTPItems(dbTPItems, itemCurations),
     ])
 
+    const totalItems = Number(allItems[0]?.total_count)
+    const concatenated = items.concat(tpItems)
```

Comment:
I'm not sure how this concat will work when we add filters to this. I think we need a function that consolidates all types of items and decides wether to do TP or Standard consolidation on them.
This also always puts third party collections last, independently on the order it got from the DB.